gui/itemStats.py

```diff
@@ -246,7 +246,7 @@ def __init__(self, parent, stuff, item):
         if not item.description:
             return
 
-        desc = item.description.replace("\n", "<br>")
+        desc = item.description.replace("\r", "<br>")
```

Comment:
Revert this please. See https://github.com/pyfa-org/Pyfa/commit/13a15a38c54b96de80788e44146a26119683375e